features/currency/managing_currency/currency_validation.feature

```diff
@@ -18,8 +18,8 @@ Feature: Currency validation
     @ui
     Scenario: Trying to remove exchange rate from existing currency
         Given the store has currency "Euro" with exchange rate 0.75
-        And I want to edit this currency
-        When I remove its exchange rate
+        When I want to edit this currency
+        And I remove its exchange rate
```

Comment:
IMO "I want to edit this currency" should belong to the "Given" steps as it's the background of the scenario.